src/routes/join/containers/SignupPage.js

```diff
@@ -1,19 +1,16 @@
-const Connect = require('react-redux').connect;
+const { useMiddleEnd } = require('strange-middle-end');
 const SignupPage = require('../components/SignupPage');
-const M = require('../../../middle-end');
 
-const internals = {};
+module.exports = function SignupPageContainer(props) {
 
-internals.connect = Connect(
-    (state) => ({}),
-    (dispatch) => ({
-        reqCreateAccount: async ( accountInfo ) => {
+    const M = useMiddleEnd();
 
-            const [err, result] = await M.dispatch.auth.createAccount(accountInfo);
+    const reqCreateAccount = async ( accountInfo ) => {
```

Comment:
Can't suggest this since it's interleaved with deleted lines, but something like this would work equally well:
```js
const reqCreateAccount = (accountInfo) => m.dispatch.auth.register(accountInfo);
```